graphql_compiler/compiler/expressions.py

```diff
@@ -641,8 +642,25 @@ def to_gremlin(self):
                              u'in Gremlin, so this is a bug. This function should not be called.')
 
     def to_cypher(self):
-        """Not implemented yet."""
-        raise NotImplementedError()
+        """Return a unicode object with the Cypher representation of this expression."""
+        self.validate()
+
+        _, field_name = self.fold_scope_location.get_location_name()
+        mark_name = u'collected_' +\
```

Comment:
Per the Google Python style guide, please use parens instead of `\` whenever possible.

```
mark_name = (
    u'collected_' + 
    emit_cypher.fold_scope_location_full_path_name(self.fold_scope_location)
)
```

Although if you write that helper function I suggested above, this would just be a call to that helper directly rather than a third separate implementation of the same naming convention.